test/unit/ngsiv2/httpBindings-test.js

```diff
@@ -0,0 +1,740 @@
+/*
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of iotagent-ul
+ *
+ * iotagent-ul is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * iotagent-ul is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with iotagent-ul.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[iot_support@tid.es]
+ *
+ * Modified by: Fernando Méndez, Daniel Calvo - ATOS Research & Innovation
+ */
+
+'use strict';
+
+var iotagentUl = require('../../../'),
+    config = require('./config-test.js'),
+    nock = require('nock'),
+    iotAgentLib = require('iotagent-node-lib'),
+    should = require('should'),
+    async = require('async'),
+    request = require('request'),
+    utils = require('../../utils'),
+    contextBrokerMock,
+    iotamMock;
+
+describe('HTTP Transport binding: measures', function() {
+    beforeEach(function(done) {
+        var provisionOptions = {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDevice1.json'),
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            }
+        };
+
+        nock.cleanAll();
+
+        iotamMock = nock('http://localhost:8082')
+        .post('/protocols')
+        .reply(200, {});
+        // Note: /v1/updateContext response is not processed by IOTA so its content is irrelevant,
+        // as far as it is a 200 OK
+        contextBrokerMock = nock('http://192.168.1.1:1026')
+        .matchHeader('fiware-service', 'smartGondor')
+        .matchHeader('fiware-servicepath', '/gardens')
+        .post('/v1/updateContext')
+        .reply(200, {});
+
+        config.iota.iotManager = {
+            host: 'localhost',
+            port: 8082,
+            path: '/protocols',
+            protocol: 'MQTT_UL',
```

Comment:
Check MQTT ocurrences in httpBinding-test.js file (3 of them at the present moment, I guess). Could be just replaced by HTTP_UL, HTTP Ultralight 2.0 IoT Agent (Node.js version), etc.?

Reply:
Fixed in 60b8e444743fa1a48df24c81fc244ea0f96d6b5a